server/backend/api/app/database.py

```diff
@@ -35,12 +35,13 @@ def get_db():
 
         db = _session_local()
         yield db
+        db.commit()
```

Comment:
Could you also do this for server/adimn_management_api. Thanks a lot!!

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/b8e308651365da07bfec0e237997b9cebc761de5